src/util/Random.h

```diff
@@ -26,16 +26,12 @@ template <typename Int>
 requires(std::is_integral_v<Int> && sizeof(Int) <= sizeof(uint64_t))
 class FastRandomIntGenerator {
  public:
-  FastRandomIntGenerator() {
+  explicit FastRandomIntGenerator(unsigned int seed = std::random_device{}()) {
```

Comment:
Implement a strong `Seed` type that wraps the unsigned int.
That way you get less confusions with the other integer arguments and maybe also with the overload resolution of the `createRandomIdTable` function.
so
`FastRandomIntGenerator(Seed seed = Seed::make(std::random_device))`
(you can maybe use our `TypedIndex` (forgot the name) class that is also used for the `VocabIndex` etc. class.

Reply:
Done, see [abbbe85](https://github.com/ad-freiburg/qlever/pull/1126/commits/abbbe85a8dd2130b9832263b6d18f9ae46d3e6a2).

The `TypeIndex` class worked perfectly and saved me a ton of time!